### PR TITLE
fix #55531 cryptopotato.com

### LINKED ISSUE #55531: cryptopotato.com
### Issue URL (Annoyance)
 [https://cryptopotato.com/omisego-gains-35-to-reach-1-in-a-week-after-launching-on-coinbase-pro-omg-price-analysis-overview/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fcryptopotato.com%2Fomisego-gains-35-to-reach-1-in-a-week-after-launching-on-coinbase-pro-omg-price-analysis-overview%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
Platform: | Win
AdGuard version: | 7.4 nightly 30
Browser: | Firefox
AdGuard driver: | TDI
Filters: | <b>Ad Blocking:</b><br/> EasyList, <br/>AdGuard Base<br/><br/><b>Annoyances:</b><br/> Web Annoyances Ultralist, <br/>Adblock Warning Removal List, <br/>I don't care about cookies, <br/>AdGuard Annoyances<br/><br/><b>Other:</b><br/> AdGuard Experimental
Custom filter subscriptions: | https://raw.githubusercontent.com/bogachenko/fuckfuckadblock/master/fuckfuckadblock.txt

### PR DESCRIPTION
**Pull request by Sergey-Lyapin**
#55531

### CODE REVIEW

File: AnnoyancesFilter/sections/annoyances.txt [hunk 1]
```diff
@@ -1392,6 +1392,7 @@ futurestud.io,slate.com,businessinsider.de,fastcompany.com,heise.de,history.com,
 bournemouthecho.co.uk,eveningtimes.co.uk,greenocktelegraph.co.uk,heraldscotland.com,swindonadvertiser.co.uk,wiltshiretimes.co.uk##.cta-component-full-width
 !
 wakacje.pl##footer[data-area="footer"] > form[action="/newsletter.html"]
+cryptopotato.com#?##sidebar section.widget:has(> div.widget-inner > h5.widget-title > span:contains(Newsletter))
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/annoyances.txt:1395]: This is "optin" widget

File: AnnoyancesFilter/sections/popups.txt [hunk 2]
```diff
@@ -92,6 +92,7 @@ sodapdf.com##.be-tooltip
 wakacje.pl#$#html.modal-active { overflow: visible!important; }
 wakacje.pl#$#html.modal-active > body { overflow: visible!important; padding-right: 0!important; }
 wakacje.pl#?#body > div[class*=" "]:has(> div[class*=" "] > div[height="auto"] > header > h3[class]:contains(/^Pomóż nam usprawnić stronę$/))
+cryptopotato.com##.mo-has-email
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/popups.txt:95]: as above -  "optin"

**Review by Alex-302 — CHANGES_REQUESTED** [review]
also missed "optin" tracker

File: AnnoyancesFilter/sections/annoyances.txt [hunk 3]
```diff
@@ -1407,6 +1407,7 @@ cointelegraph.com##.subscription-form
 ramsaydiagnostics.ru##.mailing-box
 inmyroom.ru##.c-subscribe-form
 wakacje.pl##footer[data-area="footer"] > form[action="/newsletter.html"]
+cryptopotato.com##.widget_mo_optin_widgets
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/annoyances.txt:1410]: redundant by `cryptopotato.com##.moOptinForm`

File: EnglishFilter/sections/content_blocker.txt [hunk 4]
```diff
@@ -968,6 +968,11 @@ np-coburg.de,frankenpost.de,insuedthueringen.de,augsburger-allgemeine.de,4fansit
 ! https://github.com/AdguardTeam/AdguardFilters/issues/47876
 topbusiness24.ru,mydomsam.ru,hubuhu.ru,moysadinfo.ru,mybusinessplus.ru,dukand.ru,mydizayn.ru,polzadom.ru,umnodachnik.ru,delnyesovety.ru,soverhenie.ru,mirinteresa.ru,uskorit.ru,dskyar.ru,biscand.ru,smartid24.ru,stroysaminfo.ru,topsovety.ru,sotsvetiya.ru,imdaily.ru,turizmnaok.ru,zdorovyobrazzhizni.ru,kinomann.ru,techinfolife.ru,dommoysad.ru,rteinfo.ru,psisovety.ru,kvilit.ru,speshit.ru,ollss.ru#@#.ad-placement
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/55798
+hdkinoteatr.com#@#[id*="ScriptRoot"]
```
Comment (Alex-302) [inline EnglishFilter/sections/content_blocker.txt:972]: Why hdkinoteatr.com rules are here?

File: EnglishFilter/sections/content_blocker.txt [hunk 5]
```diff
@@ -968,6 +968,11 @@ np-coburg.de,frankenpost.de,insuedthueringen.de,augsburger-allgemeine.de,4fansit
...
+@@||jsc.traffic-media.co.uk/*/hdkinoteatr.com.*.js$domain=hdkinoteatr.com
+! https://github.com/AdguardTeam/AdguardFilters/issues/55531
+cryptopotato.com##a[href="https://cryptopotato.com/bitstarz"]
```
Comment (Alex-302) [inline EnglishFilter/sections/content_blocker.txt:975]: no such element

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/content_blocker.txt (hunk 4 above)
Comment (Sergey-Lyapin) [reply EnglishFilter/sections/content_blocker.txt:972]: I don't know

File: EnglishFilter/sections/content_blocker.txt (hunk 5 above)
Comment (Sergey-Lyapin) [reply EnglishFilter/sections/content_blocker.txt:975]: It dissapeared

File: AnnoyancesFilter/sections/annoyances.txt (hunk 3 above)
Comment (Sergey-Lyapin) [reply AnnoyancesFilter/sections/annoyances.txt:1410]: If we remove this rule we leave a leftover "JOIN OUR NEWSLETTER"

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**